source/index.txt

```diff
@@ -56,6 +56,7 @@ Get Started
    Connect </connect>
    Manage Data </databases-collections>
    Explore with Playgrounds </playgrounds>
+   Test Queries </copilot-export-test>
```

Comment:
Sorry for the miscommunication here @ltran-mdb2 – I meant to put this under "Explore with Playgrounds" tab, same level as "Export to Language".

Reply:
Could we also include the word "Driver" in here? We can call it "Test Driver Queries"

Reply:
Done!